src/components/standalone/openvpn_tunnel/TunnelTable.vue

```diff
@@ -0,0 +1,218 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import NeTable from '../NeTable.vue'
+import { NeDropdown, NeButton } from '@nethserver/vue-tailwind-lib'
+import type { ServerTunnel, ClientTunnel } from './TunnelManager.vue'
+
+const { t } = useI18n()
+
+const props = defineProps<{
+  tunnels: ServerTunnel[] | ClientTunnel[]
+  isClientTunnel: boolean
+}>()
+
+const emit = defineEmits([
+  'tunnel-delete',
+  'tunnel-edit',
+  'tunnel-toggle-enable',
+  'tunnel-download'
+])
+
+const tableHeaders = [
```

Comment:
I feel like this is a bit complex, it's possible to add a some comments to explain why and what does this do? I'm feeling we're gonna lose why there's this procedural generation of the headers fields

Reply:
Added some comments to better describe the optional inclusion of headers based on the tunnel type